content/en/news/releases/1.21.x/announcing-1.21/change-notes/index.md

```diff
@@ -120,6 +120,25 @@ to conflicting inbound listeners error.
 - **Fixed** a memory leak when a `ServiceEntry` is removed.
 ([Issue #47893](https://github.com/istio/istio/issues/47893))
 
+- **Fixed** a bug when there are more than one services with same host name within the same namespace, that could occur `STRICT_DNS cluster without endpoints` error.
+([Issue #49489](https://github.com/istio/istio/issues/49489))
+
+- **Fixed** an issue occurs when using a delegate in a `VirtualService`, the effective `VirtualService` may not be consistent with expectations due to a sorting error.
+([Issue #49539](https://github.com/istio/istio/issues/49539))
+
+- **Fixed** a bug when specify a `URI` regex `.*` match within a `VirtualService` http route, it does not short circuit the other http routes behind.
+
+- **Fixed** sending stale name table when pure HTTP headless service endpoints changed.
```

Comment:
miss issue link？

Reply:
It was not there in the release notes in the PR